web/impact/impact/views/algolia_api_key_view.py

```diff
@@ -82,6 +84,20 @@ def _get_filters(request):
     if is_employee(request.user):
         return []
 
+    if request.GET['index'] == 'startup':
+        participant_roles = [UserRole.ALUM, UserRole.FINALIST, UserRole.MENTOR]
```

Comment:
something seems off here since we are setting "participant_roles to" ```[UserRole.ALUM, UserRole.FINALIST, UserRole.MENTOR]``` and which gets passed into ```_expert_specific_filter``` which by default returns the "roles" that were initially passed in; this conditional would always evaluate to true since the roles array will contain atleast the initial 3 items we populated it with: https://github.com/masschallenge/impact-api/pull/342/files#diff-44354bce8b29d5c105a3123eb9909af8R98

Reply:
I think what you want to do [here](https://github.com/masschallenge/impact-api/pull/342/files#diff-44354bce8b29d5c105a3123eb9909af8R88) is set ```participant_roles = []``` in which case the functions will populate the array as needed.

Reply:
@shanbady Let me make the recommended edits because this makes more sense.